build/tools/appveyor.bat

```diff
@@ -2,10 +2,10 @@ goto %TOOLSET%
 
 :msbuild
 cd build\msw
-msbuild /m:2 /v:n /p:Platform=%ARCH% /p:Configuration="%CONFIGURATION%" wx_vc12.sln
+msbuild /m:2 /v:n /p:Platform=%ARCH% /p:Configuration="%CONFIGURATION%" wx_vc12.sln /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
```

Comment:
Would it be possible to put this path or even the whole `/logger` option in a variable and not write it 4 times? It would also reduce the length of the line.